streaming/src/config/streaming_config.cc

```diff
@@ -37,6 +37,21 @@ void StreamingConfig::FromProto(const uint8_t *data, uint32_t size) {
   if (config.empty_message_interval() != 0) {
     SetEmptyMessageTimeInterval(config.empty_message_interval());
   }
+  if (config.flow_control_type() != proto::FlowControlType::UNKNOWN_FLOW_CONTROL_TYPE) {
```

Comment:
Please add flow_control_type in java/python too

Reply:
Flow control config have been parsed in python/java worker.